### PR TITLE
feat: [CCM-8624]: Custom Exclusion/Inclusion for AWS

### PR DESCRIPTION
**Pull request by rishik30**
#### Screenshots

Screenshot 2022-09-29 at 1 40 54 PM

Screenshot 2022-09-29 at 11 00 35 AM

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by diptiman-adak** [comment]
Quality gate Quality gate: **_Failed_** 
 


            
 #### 44 Issues 
* Code issue 44 Code issues 

 [Details](https://app.embold.io/organization/gh/harness/repositories/e864badd0b95ab81c060b15b85f15701/pull-request/10810)

**Comment by rishik30** [comment]
retrigger jest

**Comment by rishik30** [comment]
trigger cypress

File: src/modules/75-ce/components/COGatewayAccess/CustomExclusion.tsx [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+                      if (index === i) {
+                        return { ...item, value: (e.target as HTMLInputElement).value }
+                      }
+                      return item
+                    })
+                    setHandlers(updatedHandlers)
+                  }}
+                />
+                {handler.kind === HandlerKind.header && (
+                  /* istanbul ignore next */ <TextInput
```
Comment (rajarshee-harness) [inline src/modules/75-ce/components/COGatewayAccess/CustomExclusion.tsx:133]: This text input doesn't set anything?

File: src/modules/75-ce/components/COGatewayAccess/CustomExclusion.tsx [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+    debouncedSave()
+  }, [handlers, selectedType])
+
+  return (
+    <Layout.Vertical spacing={'medium'} className={cx(css.dnsLinkContainer, css.customExclusion)}>
+      <Text font={{ variation: FontVariation.H5 }}>
+        {getString('ce.co.autoStoppingRule.setupAccess.customExclusion.heading')}
+      </Text>
+      <p
+        dangerouslySetInnerHTML={{
```
Comment (rajarshee-harness) [inline src/modules/75-ce/components/COGatewayAccess/CustomExclusion.tsx:78]: Could use `String` with `useRichText`.

**Review by rajarshee-harness — APPROVED** [review]

File: src/modules/75-ce/components/COGatewayAccess/CustomExclusion.tsx (hunk 1 above)
Comment (rishik30) [reply src/modules/75-ce/components/COGatewayAccess/CustomExclusion.tsx:133]: yes this support is not yet present. Have added it anyway to make use of it in future enhancements.

File: src/modules/75-ce/components/COGatewayAccess/CustomExclusion.tsx (hunk 2 above)
Comment (rishik30) [reply src/modules/75-ce/components/COGatewayAccess/CustomExclusion.tsx:78]: both does exactly the same thing

**Review by vkbansal-harness — APPROVED** [review]

**Pull request merged**